packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { UploadTask } from '@aws-amplify/storage';
+import { DragActionHandlers } from './hooks/useFileUploader/types';
+import { ButtonProps } from 'src/primitives';
+
+export type SetShowPreviewer = (show: boolean) => void;
+type LevelInfo = 'public' | 'protected' | 'private';
+export type Files = File[];
+
+export interface UploadButtonProps extends ButtonProps {
+  className?: string;
+  disabled?: boolean;
+  onClick?: () => void;
+}
+
+export interface UploadDropZoneProps extends DragActionHandlers {
+  children?: React.ReactNode;
+  inDropZone?: boolean;
+}
+
+export interface FileUploaderProps {
+  acceptedFileTypes: string[];
+  shouldAutoProceed?: boolean;
+  hasMultipleFiles?: boolean;
+  components?: Components;
+  level: LevelInfo;
+  maxFiles?: number;
+  maxhasMultipleFilesSize?: number;
+  isPreviewerVisible?: boolean;
+  maxSize?: number;
+  onChange?: () => void;
+  onError?: (error: string) => void;
+  onSuccess?: (event: { key: string }) => void;
+  path?: string;
+  showImages?: boolean;
+  variation?: 'drop' | 'button';
+  isResumable?: boolean;
+}
+
+export interface IconProps {
+  className?: string;
+  fontSize?: string;
+}
+
+export interface PreviewerProps {
+  acceptedFileTypes: string[];
```

Comment:
This no longer needed correct? Since these types become a public contract we need to make sure we clean up anything that is no longer needed on this interface (same goes for other override components)

Reply:
ok! Will delete and double check.

Reply:
[d23a487](https://github.com/aws-amplify/amplify-ui/pull/3028/commits/d23a487464ab79fd1372e0165c1ab57616dde160)